source/get-started/index.html.md.erb

```diff
@@ -0,0 +1,95 @@
+---
+title: Get started
+weight: 11
```

Comment:
It seems a little strange to have this between the two alternate 'Install' sections – would it make sense to have this after them, like:

> GOV.UK Frontend
> Install with Node.js package manager (npm)
> Install from dist
> Get started
> Update using Node.js package manager (npm)
> Support Internet Explorer 8
> Use with old frameworks or colours
> Import CSS, assets and JavaScript
> Sass API Reference

Reply:
Good plan - will fix in the next commit.